### PR TITLE
Use edx fork of slumber, circumventing unicode decoding bug.

### PR DESCRIPTION
**Pull request by jimabramson**
XCOM-1606

@rlucioni @clintonb @feanil

### CODE REVIEW

**Comment by rlucioni** [comment]
:+1: 

Starting a new build should get you a pass now that the commit needed here is available.

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-slumber==0.7.0
+slumber==git+https://github.com/edx/slumber.git@a02e8ba381965e1de710b40af69590437adfb809#egg=slumber
```
Comment (clintonb) [inline requirements.txt]: Do we need to provide a version (e.g. `egg=slumber==0.7.0-edx`) here @feanil? If so, we will need to update `setup.py` and actually create a release on the edx/slumber repo.

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-slumber==0.7.0
+git+https://github.com/edx/slumber.git@a02e8ba381965e1de710b40af69590437adfb809#egg=slumber
```
Comment (clintonb) [inline requirements.txt]: Do we need to provide a version (e.g. egg=slumber==0.7.0-edx) here @feanil? If so, we will need to update setup.py and actually create a release on the edx/slumber repo.

**Comment by jimabramson** [comment]
also it should help that i fixed my stupid syntax mistake from the previous attempt.

File: requirements.txt (hunk 2 above)
Comment (jimabramson) [inline requirements.txt]: i think we can wait until we have more than one change on our fork, right?

**Comment by jimabramson** [comment]
updated to force install of our edx release version.   good catch @clintonb .

**Comment by rlucioni** [comment]
:+1:

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-slumber==0.7.0
+git+https://github.com/edx/slumber.git@a02e8ba381965e1de710b40af69590437adfb809#egg=slumber==0.7.0.edx
```
Comment (rlucioni) [inline requirements.txt]: Should this commit be updated? You can use the `0.7.0.edx` tag.

**Comment by clintonb** [comment]
:+1:

File: requirements.txt (hunk 3 above)
Comment (jimabramson) [inline requirements.txt]: sheesh, yes.  thanks.

**Comment by rlucioni** [comment]
@jimabramson third time's a charm. :+1:

**Comment by feanil** [comment]
:+1:

**Pull request merged**